docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +231,13 @@ test
 
 If the start page is not specified, it defaults to `about:blank`.
 
+To test webpages in local directories, you can use the `file://` protocol.
+
+```js
+fixture `MyFixture`
```

Comment:
It should be relative to current test file, e.g. if i have `.html` pages for every test and i store them in repository.